docker/android-slave/.s2i/bin/run

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
```

Comment:
We need to document that this is now main entry point. Basically little bit documentation why this is used to run .  I know that's obvious for someone who knows s2i stuff, but calling this as main entrypoint for image with source it's not standard approach and it would be good to expose why we have it. Especially:
`/usr/local/bin/run-jnlp-client "$@"`